pom.xml

```diff
@@ -210,7 +210,7 @@
              to non-snapshot public maven repos -->
         <maven.repository.protocol>https</maven.repository.protocol>
         <!-- The full remote maven repo URL; can be overridden via -D for special use cases -->
-        <maven.repository.url>${maven.repository.protocol}://repository.jboss.org/nexus/content/groups/public/</maven.repository.url>
+        <maven.repository.url>${maven.repository.protocol}://repository.jboss.org/nexus/content/groups/developer/</maven.repository.url>
```

Comment:
Is this just for testing? We can't make this change otherwise.

Reply:
This is something for test I didn't remove it.  I already fixed this . Thanks @jamezp